### PR TITLE
Converting React.createClass to ES6 classes for README

### PR DESCRIPTION
**Pull request by svenkatreddy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ module.exports = DefaultLayout;
 var React = require('react');
 var DefaultLayout = require('./layouts/default');
 
-var HelloMessage = React.createClass({
+class HelloMessage extends React.Component {
   render: function() {
```
Comment (zpao) [inline README.md]: We need to update this to just be `render() {`.

**Comment by zpao** [comment]
Overall looks good, thanks!

**Comment by zpao** [comment]
Note: I know your PR to the tutorial doing this was closed. This project is less tied to the main React communications so it's ok for us to switch now.

**Comment by svenkatreddy** [comment]
ok great, I would love to see all react repos follow a common pattern which is close to native JS / ES6 wherever possible.

**Comment by svenkatreddy** [comment]
@zpao can you merge this if everything looks good for you ?

File: README.md [hunk 2]
```diff
@@ -53,11 +53,11 @@ Your views should be node modules that export a React component. Let's assume yo
...
 var React = require('react');
 
-var HelloMessage = React.createClass({
-  render: function() {
+class HelloMessage extends React.Component {
+  render() {
     return <div>Hello {this.props.name}</div>;
   }
-});
+};
```
Comment (zpao) [inline README.md]: Sorry, missed this before - can you also remove the semicolons at the end of each of the classes?

**Comment by svenkatreddy** [comment]
@zpao Please have a look at it now.

**Pull request merged**